.github/policies/msgraph-metadata-branch-protection.yml

```diff
@@ -9,3 +9,33 @@ resource: repository
 configuration:
   branchProtectionRules:
 
+  - branchNamePattern: master
+    # This branch pattern applies to the following branches:
+    # master
+
+    # Specifies whether this branch can be deleted. boolean
+    allowsDeletions: false
+    # Specifies whether forced pushes are allowed on this branch. boolean
+    allowsForcePushes: false
+    # Specifies whether new commits pushed to the matching branches dismiss pull request review approvals. boolean
+    dismissStaleReviews: true
+    # Specifies whether admins can overwrite branch protection. boolean
+    isAdminEnforced: false
+    # Indicates whether "Require a pull request before merging" is enabled. boolean
+    requiresPullRequestBeforeMerging: true
```

Comment:
This is going to block our currently weekly generation pipeline that relies on pushing directly to master for the time being